code/game/mecha/combat/marauder.dm

```diff
@@ -44,6 +44,32 @@
 	ME = new /obj/item/mecha_parts/mecha_equipment/antiproj_armor_booster(src)
 	ME.attach(src)
 
+/obj/mecha/combat/marauder/ares
+	desc = "Heavy-duty, combat exosuit, adapted from rejected early versions of the Marauder to serve as a biohazard containment exosuit. This model, albeit rare, can be found among civilian populations."
+	name = "Ares"
```

Comment:
Might want to change this to `name = "\improper Ares"`? That'll make it show as `"That's the Ares"` rather than `"That's Ares"` when you examine it.
Just 'Ares' works fine, but I figured I'd point it out since it's improper on the wreckage.

Reply:
Every mech but the Retience doesn't have the `name ="\improper Name"`
And every mech wreckage has `name ="\improper Name wreckage"`
Did not want to make Ares the odd one out.